packages/peregrine/package.json

```diff
@@ -7,12 +7,13 @@
   "description": "The core runtime of Magento PWA",
   "main": "dist/index.js",
   "scripts": {
-    "build": "npx babel src --out-dir dist --ignore '__tests__/,__mocks__/,__fixtures__/' --source-maps --copy-files",
+    "build": "npx babel src --out-dir dist --root-mode 'upward' --ignore '__tests__/,__mocks__/,__fixtures__/' --source-maps --copy-files",
```

Comment:
Should we be using `npx` with `yarn`? Looks like [the jury is still out](https://github.com/yarnpkg/yarn/issues/3937). I'm not strictly sure it's necessary here anymore, anyway.

Reply:
The only reason to npx babel would be if we don't have it installed as a dependency of this repo.

I think if you install as a dep you can do away with `npx` here. I guess it just depends on the reason.